### PR TITLE
Add new connection keycodes for Bluetooth, 2.4GHz.

### PR DESCRIPTION
**Pull request by tzarc**
## Description

As discussed on Discord.

## Types of Changes

- [x] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Review by KarlK90 — APPROVED** [review]
LGTM

**Review by drashna — APPROVED** [review]

File: quantum/keycodes.h [hunk 1]
```diff
@@ -1365,9 +1393,6 @@ enum qk_keycode_defines {
     SC_LAPO    = QK_SPACE_CADET_LEFT_ALT_PARENTHESIS_OPEN,
     SC_RAPC    = QK_SPACE_CADET_RIGHT_ALT_PARENTHESIS_CLOSE,
     SC_SENT    = QK_SPACE_CADET_RIGHT_SHIFT_ENTER,
-    OU_AUTO    = QK_OUTPUT_AUTO,
```
Comment (zvecr) [inline quantum/keycodes.h:1368]: Could we add to `quantum/quantum_keycodes_legacy.h`

```c
#define QK_OUTPUT_AUTO OU_AUTO
```

Comment (tzarc) [reply quantum/keycodes.h:1368]: Done.

**Comment by tzarc** [comment]
Will write up docs now that we've got sufficient acceptance of direction.

**Pull request merged**